adhocracy-plus/templates/a4dashboard/project_list.html

```diff
@@ -9,7 +9,7 @@ <h1 class="lr-bar__left">
             {% trans 'Projects' %}
         </h1>
         <div class="lr-bar__right">
-            <a href="{% url 'a4dashboard:project-create' organisation_slug=view.organisation.slug %}" class="btn btn--light">
+            <a href="{% url 'a4dashboard:project-create' organisation_slug=view.organisation.slug %}" class="btn btn--default" disabled>
```

Comment:
Why disabled?

Reply:
@fuzzylogic2000 I think I removed it in another commit though as can't find it now?!

Reply:
But it's still here also when I look at all changed files and not at single commits. And I saw it in the inspector when I tried it out (even though it didn't do anything).
But I can remove it after lunch - still have it checked out anyway.

Reply:
@fuzzylogic2000 sorry hadn't pushed my latest :(